crates/ruff/src/rules/dlint/snapshots/ruff__rules__dlint__tests__DUO119_DUO119.py.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/ruff/src/rules/dlint/mod.rs
+---
+DUO119.py:1:8: DUO119 Use of the `shelve` module should be avoided
+  |
+1 | import shelve
+  |        ^^^^^^ DUO119
+  |
+
+DUO119.py:4:20: DUO119 Use of the `shelve` module should be avoided
+  |
+4 | from shelve import open, Shelf
```

Comment:
The range here looks wrong. Are we now highlighting the import name rather than the module name?

Reply:
Ah my bad. Misunderstood what you meant by https://github.com/charliermarsh/ruff/pull/4482#discussion_r1198625178 . Will fix

Reply:
Hmm, I forgot about that. No, I don't think there is. At least, not before we include range information for `Identifier` too (by making it a proper `Node`. 

We have a specific helper for classes and functions to extract the identifier name. We could potentially do something similar for `StmtImportFrom` or we go back to highlighting the whole statement, since we can't really do better now

https://github.com/charliermarsh/ruff/blob/45fc49773753388984f65dfa0f54bc71cf779a41/crates/ruff_python_ast/src/helpers.rs#L1092-L1108

Reply:
Addded a helper using almost identical logic and it seems to do the trick! Could be folded into `identifier_range` due to almost identical logic but was unsure whether to keep separate for functional scope (i.e `ClassDef`, `FunctionDef` and `AsyncFunctionDef` are a sensible grouping. Adding a `StmtImportFrom` less so). Left it separate for now